### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by ntzwilly**
# Project 3: Using Bootstrap - Milestone 1 

## Description: 

1. Set up a new repository
2.  Prepare the main HTML file
3.  Install Bootstrap in your project
4.  Create these sections of the website for the 3 screen sizes: Header and Footer.

### CODE REVIEW

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Invalid 🛑 

Hello, you've made great progress, but I can't review your work unless the requirements below are met 👇 

## do NOT move to the next milestone, fix the issue below, and ask for another review.

- [x] The project should have a descriptive README.md ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Comment by ntzwilly** [comment]
# Project 3: Using Bootstrap - Milestone 1

## Description:

1. Set up a new the repository
2. Prepare the main HTML file
3. Install Bootstrap in your project
4. Prepare your project to work with 3 screen sizes: mobile, tablet, and desktop using the Bootstrap
5. Responsive grid classes
6. Create these sections of the website for the 3 screen sizes

File: style.css [hunk 1]
```diff
@@ -0,0 +1,170 @@
+
+.container-fluid {
+  max-width: 1310px;
+  background-color: #fff;
```
Comment (jubaan) [inline style.css]: - [x] Bootstrap

File: style.css [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+}
+
+.navbar-top {
+  min-height: 50px;
+  font-size: 17px;
+  background-color: #f72210;
+  color: #fff;
```
Comment (jubaan) [inline style.css]: - [x] Bootstrap

File: style.css [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+}
+
+#btn-login {
+  font-weight: bold;
+  display: inline;
```
Comment (jubaan) [inline style.css]: - [x] Bootstrap

File: style.css [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+  margin-left: 85px;
+  margin-top: 2px;
+}
+
+#btn-sub {
+  border-radius: 20px;
+  height: 29.6px;
+  width: 127px;
+  border: none;
+  font-weight: bold;
```
Comment (jubaan) [inline style.css]: - [x] Bootstrap

File: style.css [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+  font-family: "Roboto Condensed", sans-serif;
+}
+
+.burger-menu {
+  margin-left: 15px;
+}
+
+.burger {
+  cursor: pointer;
+  display: block;
```
Comment (jubaan) [inline style.css:304]: - [x] Bootstrap

File: style.css [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+  margin-bottom: 4px;
+  height: 4px;
+  width: 30px;
+  background-color: #ffffff8c;
+}
+
+.top-list {
+  list-style-type: none;
+  display: inline;
+  padding: 0;
```
Comment (jubaan) [inline style.css]: - [x] Bootstrap

File: style.css [hunk 7]
```diff
@@ -0,0 +1,170 @@
...
+  border-left-width: 1px;
+  border-left-style: solid;
+  border-left-color: #d9d9d9;
+  padding-left: 10px;
+  padding-right: 5px;
+}
+
+.no-border {
+  border: none;
+  padding-left: 0;
```
Comment (jubaan) [inline style.css]: - [x] Bootstrap

File: style.css [hunk 8]
```diff
@@ -0,0 +1,170 @@
...
+.bottom-list {
+  list-style-type: none;
+  display: inline;
+  padding: 0;
+  width: 80%;
+  margin-left: 15px;
+}
+
+.nav-item {
+  font-weight: bold;
```
Comment (jubaan) [inline style.css]: - [x] Bootstrap

File: style.css [hunk 9]
```diff
@@ -0,0 +1,170 @@
...
+  font-size: 14px;
+  display: inline;
```
Comment (jubaan) [inline style.css]: - [ ] Bootstrap

File: style.css [hunk 10]
```diff
@@ -0,0 +1,170 @@
...
+  padding-left: 10px;
+  padding-right: 10px;
+}
+
+:hover.red-hover {
+  color: #f72210;
+}
+
+:hover.orange-underline {
+  text-decoration: underline;
```
Comment (jubaan) [inline style.css]: - [ ] Bootstrap

File: style.css [hunk 11]
```diff
@@ -0,0 +1,170 @@
...
+  text-decoration-color: #f72210;
+}
+
+.grey {
+  color: grey;
+}
+
+.bold {
+  font-weight: bold;
+}
```
Comment (jubaan) [inline style.css]: - [ ] Bootstrap

File: style.css [hunk 12]
```diff
@@ -0,0 +1,170 @@
...
+
+.main-title-lg {
+  font-size: 76px;
+  height: 95px;
+  font-family: monospace, arial, sans-serif;
+  font-weight: bold;
```
Comment (jubaan) [inline style.css]: - [ ] Bootstrap

File: style.css [hunk 13]
```diff
@@ -0,0 +1,170 @@
...
+}
+
+.main-title-md {
+  font-size: 23px;
+  font-family: monospace, arial, sans-serif;
+  font-weight: bold;
```
Comment (jubaan) [inline style.css]: - [ ] Bootstrap

File: style.css [hunk 14]
```diff
@@ -0,0 +1,170 @@
...
+}
+
+.footer-logo-large {
+  color: #f72210;
+  font-size: 20px;
+  font-weight: bold;
```
Comment (jubaan) [inline style.css]: - [ ] Bootstrap

File: style.css [hunk 15]
```diff
@@ -0,0 +1,170 @@
...
+
+.footer-two {
+  border-right: 1px solid rgba(0, 0, 0, 0.2);
+  padding-right: 10px;
+  padding-left: 10px;
+  font-size: 15px;
+}
+
+.footer-two:hover {
+  color: black;
```
Comment (jubaan) [inline style.css:834]: - [ ] Bootstrap

File: style.css [hunk 16]
```diff
@@ -0,0 +1,170 @@
...
+  list-style-type: none;
+  display: inline;
+  padding: 0;
+  margin-top: 10px;
+  max-width: 95%;
+}
+
+.items {
+  border-left: 1px solid black;
+  font-weight: bold;
```
Comment (jubaan) [inline style.css]: - [ ] Bootstrap

File: index.html [hunk 17]
```diff
@@ -0,0 +1,116 @@
...
+              <span class = "footer-two">Contact Us</span>
+              <span class = "footer-two">Editorial Guidelines</span>
+              <span class = "footer-two">Advertise</span>
+              <span class = "footer-two">Copyright</span>
+              <span class = "footer-two">Terms & Conditions</span>
+              <span class = "footer-two">Privacy Policy</span>
+              <span class = "footer-two">Cookie Policy</span>
+              <span class = "footer-two">Terms of Sale</span>
+              <span class = "footer-two">Archive</span>
+              <span class = "footer-two">Announcements</span>
```
Comment (jubaan) [inline index.html]: - [ ] Use semantic HTML for the links `<a>`

File: index.html [hunk 18]
```diff
@@ -0,0 +1,116 @@
...
+        </div>
+        <hr>
+          <div class = "footer-one">
+            <div class = "footer-edition">
+              <span><b>Editions:</b></span>
+              <span class = "footer-two">U.S. Edition</span>
+              <span class = "footer-two">日本</span>
+              <span class = "footer-two">Pakistan</span>
+              <span class = "footer-two">Polska</span>
+              <span class = "footer-two">România</span>
```
Comment (jubaan) [inline index.html]: - [ ] Use semantic HTML for the links `<a>`

File: index.html [hunk 19]
```diff
@@ -0,0 +1,116 @@
...
+          <div>
+            <ul class = "bottom-list d-flex justify-content-sm-between">
+              <li class = "nav-item">TRENDING</li>
+              <li class = "nav-item grey red-hover">JOE BIDEN</li>
+              <li class = "nav-item grey red-hover">DONALD TRUMP</li>
+              <li class = "nav-item grey red-hover">INAUGURATION</li>
+              <li class = "nav-item grey red-hover">CORONAVIRUS</li>
+              <li class = "nav-item grey red-hover">STIMULUS PACKAGE</li>
+              <li class = "nav-item grey red-hover">VACCINE</li>
+              <li class = "nav-item grey red-hover">CAPITOL RIOTS</li>
```
Comment (jubaan) [inline index.html]: - [x] Use semantic HTML for the links `<a>`

File: index.html [hunk 20]
```diff
@@ -0,0 +1,116 @@
...
+              <li class = "items red-hover">Culture</li>
+              <li class = "items red-hover">Newsgeek</li>
+              <li class = "items red-hover">Sports</li>
+              <li class = "items red-hover">Health</li>
+              <li class = "items red-hover">The Debate</li>
+              <li class = "items red-hover">Vantage</li>
+              <li class = "search-bar">
+                Search
+                <img id = "search-icon" alt="search-image" src = "https://g.newsweek.com/www/images/icon-search-glass.svg">
+              </li>
```
Comment (jubaan) [inline index.html]: - [x] Should use a form with a search input

File: index.html [hunk 21]
```diff
@@ -0,0 +1,116 @@
...
+              <li class = "no-border items red-hover">U.S.</li>
+              <li class = "items red-hover">World</li>
+              <li class = "items red-hover">Business</li>
+              <li class = "items red-hover">Tech & Science</li>
+              <li class = "items red-hover">Culture</li>
+              <li class = "items red-hover">Newsgeek</li>
+              <li class = "items red-hover">Sports</li>
+              <li class = "items red-hover">Health</li>
+              <li class = "items red-hover">The Debate</li>
+              <li class = "items red-hover">Vantage</li>
```
Comment (jubaan) [inline index.html]: - [x] Use semantic HTML for the links `<a>`

File: index.html [hunk 22]
```diff
@@ -0,0 +1,116 @@
...
+            <div class = "main-title-lg d-none d-lg-block">
+              Newsweek
+            </div>
+            <div class = "main-title-md d-block d-lg-none">
+              Newsweek
+            </div>
+          </div>
+          <div class = "col-lg-4 login-sub col-6 text-lg-end d-flex justify-content-end">
+            <div id = "btn-login">LOGIN</div>
+            <span><button id = "btn-sub">SUBSCRIBE ></button></span>
```
Comment (jubaan) [inline index.html]: - [x] Use semantic HTML for the links `<a>`

File: index.html [hunk 23]
```diff
@@ -0,0 +1,116 @@
...
+          <div class = "col-lg-4 time d-none d-lg-block">
+            Wed, Jan 20, 2021 
+          </div>
+          <div class = "col-lg-4 col-6 text-lg-center">
+            <div class = "main-title-lg d-none d-lg-block">
+              Newsweek
+            </div>
+            <div class = "main-title-md d-block d-lg-none">
+              Newsweek
+            </div>
```
Comment (jubaan) [inline index.html]: - [x] Code is duplicated, use Bootstrap classes with media query syntax to accomplish the same

File: index.html [hunk 24]
```diff
@@ -0,0 +1,116 @@
...
+<body>
+  <header>
+    <div class = "container-fluid">
+      <div class = "navbar-top row align-items-center"> 
+          <div class = "col-lg-4 time d-none d-lg-block">
+            Wed, Jan 20, 2021 
+          </div>
+          <div class = "col-lg-4 col-6 text-lg-center">
+            <div class = "main-title-lg d-none d-lg-block">
+              Newsweek
```
Comment (jubaan) [inline index.html]: - [x] Newsweek in header and footer is a link also. Use an anchor tag for both of them `<a>`

**Review by jubaan — CHANGES_REQUESTED** [review]

**Comment by ntzwilly** [comment]
# Project 3: Using Bootstrap - Milestone 1

## Description:

1. Set up a new the repository
2. Prepare the main HTML file
3. Install Bootstrap in your project
4. Prepare your project to work with 3 screen sizes: mobile, tablet, and desktop using the Bootstrap
5. Responsive grid classes
6. Create Header and Footer of the website for the 3 screen sizes

## Corrected all code review feedbacks

**Review by SpaYco — APPROVED** [review]
# Project Approved ✔️ 

Sorry for the confusion, milestone 1 and 2 should always be approved, but seems that the previous TSE marked it by mistake to "changes requested" 
you can go ahead and move to the next milestone while focusing on what the previous TSE said.

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Comment by ntzwilly** [comment]
# Project 3: Using Bootstrap - Milestone 1

In the second milestone, I create the 3 columns main section of the website for the 3 screen sizes.

## Column 1

Featured stories
Sponsored insights / Rankings

## Column 2

Top story
My turn
Culture & travel
More stories
Latest news

## Column 3

The debate Opinion Sponsored insights / Rankings Free signup

**Review by SpaYco — APPROVED** [review]
# Project Approved ✔️ 

Milestone approved, however, there are things that you need to change, you can either make the changes now, or you keep them to the last milestone. but keep in mind **all these changes unless marked optional will be MANDATORY in the third milestone**.

- [x] the 'Sponsored Insight' should be blue.
- [x] you have a lot of CSS properties that can be turned into Bootstrap classes, it's recommended to fix those from now as it's mandatory for them to be fixed in order for the project to be approved in Milestone 3.

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Comment by ntzwilly** [comment]
# Newsweek-replica

I was required to complete the milestones below to learn bootstrap. This project consists of replicating the Newsweek website putting emphasis on how it behaves differently depending on the size of the screen where the website is rendered.

## Below are the major concepts I learned:

- Ability to create UIs adaptable to different screen sizes using media queries
- Use a CSS framework like Bootstrap
- Apply front-end best practices
- Create UIs adjusted to given designs
- Understand the usefulness of transitions and animations

## Project 3: Using Bootstrap - Milestone 1

- Set up a new the repository
- Prepare the main HTML file
- Install Bootstrap in your project
- Prepare your project to work with 3 screen sizes: mobile, tablet, and desktop using the Bootstrap responsive grid classes
- Create these sections of the website for the 3 screen sizes

## Milestone 2 

In the second milestone, you will create the 3 columns main section of the website for the 3 screen sizes.

## Column 1

- Featured stories
- Sponsored insights / Rankings

## Column 2

- Top story
- My turn
- Culture & travel
- More stories
- Latest news

## Column 3

- The debate
- Opinion
- Sponsored insights / Rankings
- Free signup

## Milestone 3 - complete mobile version

The third and last milestone of the project is to complete the last sections of the website and prepare it for my portfolio.

**Review by PhillipUg — CHANGES_REQUESTED** [review]
Hey team, 👋

You have done an amazing job implementing the requirements for this project.

However, there are still some changes 👇 you need to implement to approve this request 🔍

### Changes Required ♻️ 

- [x] All the images of the `More stories`, `Latest News` and the `In the Magazine` sections and all the other images that follow below should have links.
- [x] The navigation items at the bottom in the footer should all have links.
- [x] All the titles in the different sections should use hx tags (h1 - h6). 
- [x] Newsletter section at the bottom should have an input with type email

#### Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack as **Phillip Musiime** or
 <a href="mailto:phillipmusiime@gmail.com"> gmail</a> &nbsp; &nbsp; [Twitter Follow](https://twitter.com/intent/follow?screen_name=Phillip_Ug) &nbsp; &nbsp; [GitHub followers](https://github.com/PhillipUg)

**Comment by ntzwilly** [comment]
## Thank you for your feedback. I have made the required changes:

- All the images of the More stories, Latest News and the In the Magazine sections and all the other images that follow below should have links.
- The navigation items at the bottom in the footer should all have links.
- All the titles in the different sections should use hx tags (h1 - h6).
- Newsletter section at the bottom should have an input with type email

**Review by SpaYco — APPROVED** [review]
# Project Approved ✔️ 

thank you for making the required changes, your project is now approved!

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Pull request merged**